sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -26,16 +26,17 @@ export interface BeginRecoverDeletedKeyOptions extends KeyPollerOptions {
 
 // @public
 export interface CreateEcKeyOptions extends CreateKeyOptions {
-    curve?: KeyCurveName;
-    hsm?: boolean;
+    curve?: KeyCurveName; // Should we rename to curveName and deprecate curve?
+    hsm?: boolean; // Should we rename to hardwareProtected and deprecate hsm (breaking change)?
```

Comment:
Same as above, but you could always document that this means "Hardware Security Module".